src/actions/base.ts

```diff
@@ -38,6 +38,14 @@ export let compareKeypressSequence = function(one: string[] | string[][], two: s
     );
   };
 
+  const isSingleAlpha = (s: string): boolean => {
+    if (s.length === 1 && (('a' <= s && s <= 'z') || ('A' <= s && s <= 'Z'))) {
```

Comment:
Regex? ^[a-zA-Z]$

Reply:
Yeah!.
Should I use regex here?
isSingleNumber does not use regex, so I adapted it.

Reply:
`isSingleNumber` could be regex too, but it's a little bit simpler so I'm more ambivalent.

Reply:
I fix it.
1cb61f1f724e058e51774d25acfaf18b4525ef29